source/codegen/metadata/nixnet/functions.py

```diff
@@ -389,7 +390,8 @@
         'returns': 'nxStatus_t'
     },
     'DbCreateObject': {
-        'codegen_method': 'no',
+        'init_method': True,
+        'custom_close': "", # we don't need to close reference from this method
```

Comment:
await: We would ideally like all of these to close when the database is closed right? We actually have a mechanism for that with `add_dependent_session`.

```python
db = stub.open_database(...)
table = stub.create_object(db, "table", ...)
row = stub.create_object(table, "row", ...)
stub.close_database(db)
```

Do we want that last line to clean up table and row? If so, I think we can do that with `add_dependent_session`. Each db_object would be a dependent session of the parent_object_ref and it would cascade down.

In the current implementation, is there any way to directly close the sessions with `custom_close=""`? They would be  cleaned up with `reset_server` (that's good!) but there's no way to clean them up individually.

Reply:
Last line does cleanup table and row from driver side, but I had missed removing them from session repository. `DbCreatedObject` has equivalent `DbDeleteObject` that can be used as `custom_close` here. I have fixed function metadata to do that.